pkg/services/datasources/service/cache_service.go

```diff
@@ -1,27 +1,27 @@
-package datasources
+package service
 
 import (
 	"context"
 	"fmt"
 	"time"
 
 	"github.com/grafana/grafana/pkg/infra/localcache"
+	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/models"
 	"github.com/grafana/grafana/pkg/services/sqlstore"
 )
 
+var (
```

Comment:
how about remove the global logger also in the PR. I know it is not related to this :D